params/config.go

```diff
@@ -113,6 +115,8 @@ type ChainConfig struct {
 
 	FeeConfig          *FeeConfig `json:"feeConfig,omitempty"`
 	AllowFeeRecipients bool       `json:"allowFeeRecipients,omitempty"` // Allows fees to be collected by block builders.
+
+	AllowListConfig precompile.AllowListConfig `json:"allowListConfig"` // Config for the allow list precompile
```

Comment:
If this is `omitempty` it will still instantiate as an empty var...I'd suggest we do that for consistency.

Reply:
Sounds good